cve_bin_tool/input_engine.py

```diff
@@ -31,17 +32,17 @@ def __new__(cls, value, *aliases):
 
 
 class InputEngine:
-    parsed_data: Dict[tuple, Remarks]
+    parsed_data: DefaultDict[tuple, Dict[str, Dict[str, Any]]]
 
     def __init__(
         self, filename: str, logger: Logger = None, error_mode=ErrorMode.TruncTrace
     ):
         self.filename = os.path.abspath(filename)
         self.logger = logger or LOGGER.getChild(self.__class__.__name__)
         self.error_mode = error_mode
-        self.parsed_data = {}
+        self.parsed_data = defaultdict(dict)
 
-    def parse_input(self) -> Dict[tuple, Remarks]:
+    def parse_input(self) -> DefaultDict[tuple, Dict[str, Dict[str, Any]]]:
```

Comment:
Another place where `remarks` told us a lot more than Dict[str, Dict[str, any]] does.  Should we be using something like `triage_data` to replace remarks for clarity?

Reply:
Yes, We can construct our own type. TriageData and that will solve problem.